src/libasr/codegen/llvm_utils.cpp

```diff
@@ -587,4 +594,43 @@ namespace LFortran {
         }
     }
 
+    llvm::Value* LLVMTuple::check_tuple_equality(llvm::Value* t1, llvm::Value* t2,
```

Comment:
This needs to go into a more generic function I believe.

Reply:
I will do that in the next commit, so that list of lists and tuple with list work well.